pytest_pootle/fixtures/models/user.py

```diff
@@ -12,6 +12,18 @@
 from pytest_pootle.env import TEST_USERS
 
 
+@pytest.fixture(
+    scope="session",
+    params=["nobody", "admin", "member", "member2"])
+def request_users(request):
+    from django.contrib.auth import get_user_model
+
+    info = TEST_USERS[request.param]
```

Comment:
There is a similar case below where you copy dictionary first. Is it ok to edit `TEST_USERS` here?